packages/lodestar/src/api/impl/node/index.ts

```diff
@@ -51,13 +51,34 @@ export function getNodeApi(opts: IApiOptions, {network, sync}: Pick<ApiModules,
     },
 
     async getPeerCount() {
-      // TODO: Implement
+      let disconnected = 0;
+      let connecting = 0;
+      let connected = 0;
+      let disconnecting = 0;
+
+      for (const connections of network.getConnectionsByPeer().values()) {
+        for (const connection of connections) {
+          switch (connection.stat.status) {
+            case "open":
+              connected++;
+              break;
+            case "closing":
+              disconnecting++;
+              break;
+            case "closed":
+              disconnected++;
+              break;
+            default:
+              connecting++;
```

Comment:
So if we have 1 peer with 20 open connections it will return `connecting: 20`, is this the expected behavior?

Reply:
updated to use the most relevant connection for a peer in the counting